app/assets/javascripts/controllers/cloud_topology/cloud_topology_controller.js

```diff
@@ -0,0 +1,278 @@
+/* global miqHttpInject */
+
+miqHttpInject(angular.module('cloudTopologyApp', ['kubernetesUI', 'ui.bootstrap', 'ManageIQ']))
+.controller('cloudTopologyController', CloudTopologyCtrl);
+
+CloudTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService'];
+
+function CloudTopologyCtrl($scope, $http, $interval, $location, topologyService) {
```

Comment:
this format is ok, the last hash doesn't need {}, this is just passed as an argument for rails .includes